entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+set -e
+
+testConnection="/opt/mssql-tools/bin/sqlcmd -S sql -U sa -P Password123"
+
+until $testConnection; do
+>&2 echo "SQL Server is starting up.."
+sleep 1
+done
```

Comment:
updated please review! travis_retry is used when you install dependencies within travis and it gives timeout. in this case we install dependencies within docker